src/cmd/cmd.cpp

```diff
@@ -388,24 +398,20 @@ int main(int argc, char **argv)
         }
     }
 
-    // take the unmodified url to pass to csync_create()
-    QByteArray remUrl = options.target_url.toUtf8();
-
     // Find the folder and the original owncloud url
-    QStringList splitted = url.path().split("/" + account->davPath());
```

Comment:
why do you remove this instead of fixing it such that the value returned by davPath() is valid ?